AppCenterDistribute/AppCenterDistribute/Model/MSACReleaseDetails.h

```diff
@@ -37,7 +38,7 @@
 /**
  * The flag that indicates whether the release is a mandatory update or not.
  */
-@property(nonatomic, getter=isMandatoryUpdate, assign) BOOL mandatoryUpdate;
+@property(nonatomic, getter=isMandatoryUpdate, assign) BOOL mandatoryUpdate NS_SWIFT_NAME(mandatoryUpdate);
```

Comment:
Probably we can take away `NS_SWIFT_NAME(mandatoryUpdate)`, as the property already has the same name.